### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by RetroAech** [maruohon/tweakeroo#9]
https://paste.ee/p/evNPn

**Comment by maruohon** [comment]
Looks like the Amecs mod is crashing in one of their Mixins.
Does this crash not happen *just* without Tweakeroo? That doesn't really make sense, as Tweakeroo does not even touch the same class.

**Comment by RetroAech** [comment]
Yeah I never took Amecs out just Tweakeroo! Sounds curious! Any advice for testing?

**Comment by maruohon** [comment]
So just taking out Tweakeroo fixes it? That's very strange, is that the correct crash log then? How about leaving in Tweakeroo but taking out Amecs, does the game then launch?

**Comment by maruohon** [comment]
Well at least Amecs and all my mods work together just fine, so it must be some interaction with some other mods as well. So you would just have to try taking out mods until you find one that causes it.

![2019-07-23_21 38 26](https://user-images.githubusercontent.com/2331050/61738359-c8f76e00-ad92-11e9-8540-16d4606597b1.png)

**Comment by maruohon** [comment]
Any update on this? Looks like the paste is also gone already...

**Comment by Styro2017** [comment]
I was having a similar issue, where removing tweakeroo solves a crash on startup with the exit code 0.  

Here is my current mod list:
https://gyazo.com/8704a0a0114e81d6ca00636cc8472b65

Let me know if there is anything I should try.

**Comment by maruohon** [comment]
@Styro2017 That's probably the crash/conflict with FabricCarpet I fixed yesterday.
Get the latest Tweakeroo from here: https://masa.dy.fi/mcmods/client_mods/?mcver=1.16.4
